test/IntegrationTests/MSTest.Acceptance.IntegrationTests/Program.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+using Microsoft.Testing.Extensions;
+using Microsoft.Testing.Framework.Configurations;
+using Microsoft.Testing.Platform.CommandLine;
+#if ENABLE_CODECOVERAGE
+using Microsoft.Testing.Platform.Extensions.CodeCoverage;
+#endif
```

Comment:
We don't have the flag in this repo

```suggestion
using Microsoft.Testing.Extensions.CodeCoverage;
```

Reply:
I'll follow-up in next PR